cards/tasks.py

```diff
@@ -72,7 +72,7 @@ def update_cards_with_balance(balance_spreadsheet):
                 card.profile_id = int(row.get('cardholderid'))
                 card.save()
                 updated_count += 1
-            except CardBalance.DoesNotExist:
+            except (CardBalance.DoesNotExist, CardBalance.MultipleObjectsReturned):
```

Comment:
I mean migration fixes that right? So in case of MultipleObjectsReturned we would like to escalate